### PR TITLE
Intergrating Airtable API with Next.js serverless functions

### LINKED ISSUE #2955: Integrating Airtable API with Next.js serverless functions
### Proposed title of article
Integrating Airtable API with Next.js serverless functions

### Outline
In this article, I'm going to discuss how to integrate Airtable JavaScript API into React applications. Airtable is a spreadsheet-database hybrid, with the features of a database but applied to a spreadsheet. Next.js is a React framework which supports server-side programming without using any other JavaScript Frameworks. We are going to build a simple **REST** API for storing a form data to Airtable database using Next.js serverless functions.

### Key takeaways: 
By the end of this article, one can understand about:
- what are **JAM** applications and how to build one.
- what are serverless functions how to use them.
- How to build and deploy web applications using Next.js.
- How to integrate Airtable JavaScript API with web applications.

### PR DESCRIPTION
**Pull request by giridhar7632**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)(2766)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**(anyone new to using Airtable can understand perfectly)

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images(I added my own image)
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**(3%)
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)(Grade 5)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?(I tried to rebuild the whole application myself after completing the article)
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?(I have added the source code and live demo for the application)

This closes #2955

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept. Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content. Be sure to see our resources page to see more info on plagiarism and what is considered as such.
[Intergrating Airtable API with Next.js serverless functions #3487.pdf](https://github.com/section-engineering-education/engineering-education/files/7072316/Intergrating.Airtable.API.with.Next.js.serverless.functions.3487.pdf)

**Comment by giridhar7632** [comment]
hey @WanjaMIKE most of the plagiarism error was because of the code. Both the examples are todo apps built using next.js and tailwind css. Should I change the example in my article?

**Comment by giridhar7632** [comment]
Hey @katungi I have changed the content acc to the suggestions by @WanjaMIKE 
Please review it

File: content/articles/integrate-airtable-with-nextjs/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-![Hero image](/hero-img.png)
+![Hero image](https://cloud-ed45a3xcq-hack-club-bot.vercel.app/1hero-img.png)
```
Comment (katungi) [inline content/articles/integrate-airtable-with-nextjs/index.md:1]: Hey @giridhar7632. Section does not allow cdn images. We prefer all the images hosted locally. 
Please revert that change. Please refer to the [docs](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md) and fix image links.

Finally, please run your article through a grammar checker like [Grammarly](https://www.grammarly.com/) to improve the quality of the article.

**Comment by WanjaMIKE** [comment]
Hi @giridhar7632 several issues were raised during the review process. Could you work more on the article, specifically, fix grammar errors, sentence structure, formatting  - then open a new PR when its really ready. Thanks

**Review by katungi — CHANGES_REQUESTED** [review]
Please fix those changes mentioned in the comments. At Section we have a desired quality for our articles. Your article needs a lot of grammar changes and revisions to sentence structure and it would be easier if you fixed them first before we move on. 

Feel free to ask for any clarification from the community. We are all here to help you produce the best article possible.

**Pull request closed without merging**